frontends/web/src/locales/en/app.json

```diff
@@ -600,6 +600,19 @@
       "title": "How to restore a wallet from a backup"
     }
   },
+  "success": {
```

Comment:
Doesn't matter much, but why did you move the whole section up? It will be move down again when I pull from locize.

Reply:
oh noes